spark2.3/build-assembly.sbt

```diff
@@ -55,6 +56,7 @@ assemblyExcludedJars in assembly := {
     // Included in / Conflicts with
     // jcl-over-slf4j-1.7.5.jar
     "commons-logging-1.1.1.jar",
+    "commons-lang3-3.6.jar",
```

Comment:
How does this help "fix imports for build"? Why is this needed?

Reply:
`sbt assembly` started giving errors like

`[error] deduplicate: different file contents found in the following:`
`[error] ~/.ivy2/cache/org.apache.commons/commons-lang3/jars/commons-lang3-3.6.jar:org/apache/commons/lang3/tuple/Triple.class`
`[error] ~/.m2/repository/com/splicemachine/utilities/3.2.0.1982-SNAPSHOT/utilities-3.2.0.1982-SNAPSHOT.jar:org/apache/commons/lang3/tuple/Triple.class`

apparently as a result of new shading in jars in the spliceengine project.